nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestUnpackContent.java

```diff
@@ -222,6 +225,88 @@ public void testInvalidZip() throws IOException {
             flowFile.assertContentEquals(path.toFile());
         }
     }
+    @Test
+    public void testZipEncodingField() {
```

Comment:
Super wonderful that you created a unit test to cover this and provided a simple example zip to do so!  But the checks will likely fail because the zip file doesn't have a license header so I believe you'll need to add it to an excludes list for the RAT process to not try to extract a header.  You'll see other examples in some pom files for such exclusions.